### PR TITLE
further cleanup and documentation

### PR DESCRIPTION
**Pull request by paulr34**

### CODE REVIEW

File: src-electron/sdk/matter.js [hunk 1]
```diff
@@ -110,17 +106,29 @@ async function computeStorageOptionNewConfig(storagePolicy) {
   }
   return storageOption
 }
-
+/**
+ * This asynchronous function computes and returns the new configuration for a storage policy.
+ *
+ * @param {Object} db - The database instance.
+ * @param {Object} clusterRef - The reference to the cluster.
```
Comment (brdandu) [inline src-electron/sdk/matter.js:113]: clusterRef -> Number?

Comment (paulr34) [reply src-electron/sdk/matter.js:113]: hm. you're right

File: src-electron/sdk/matter.js [hunk 2]
```diff
@@ -110,17 +106,29 @@ async function computeStorageOptionNewConfig(storagePolicy) {
...
   clusterRef,
   storagePolicy,
   forcedExternal,
   attributeName
 ) {
-  let clusterName
-  clusterName = await queryCluster.selectClusterName(db, clusterRef)
-  forcedExternal.map((option) => {
+  let clusterName = await queryCluster.selectClusterName(db, clusterRef)
+  forcedExternal.some((option) => {
```
Comment (tecimovic) [inline src-electron/sdk/matter.js:130]: I don't get this .some(). Why are we using .some()? 
why are we not doing `for ( ... of ...)` ?

Comment (tecimovic) [reply src-electron/sdk/matter.js:130]: If we are using .some(), we should properly return a value back.

**Review by tecimovic — APPROVED** [review]
Some minor issues, but generally fine....

**Review by brdandu — APPROVED** [review]

**Pull request merged**